sassc/plan.sh

```diff
@@ -0,0 +1,26 @@
+pkg_origin=core
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_name=sassc
+pkg_version=3.3.6
+pkg_source=https://github.com/sass/${pkg_name}/archive/${pkg_version}.tar.gz
+pkg_shasum=3d87edeacdd0f9a21cd0bdcf0e0e40c832b21f469af80eb5e11488c66bf53840
+pkg_license=('MIT')
+pkg_description=('libsass command line driver')
+pkg_upstream_url=https://github.com/sass/sassc
+pkg_deps=(core/glibc core/gcc)
+pkg_build_deps=(core/make core/wget)
+pkg_bin_dirs=(bin)
+pkg_lib_dirs=(lib)
+
+do_prepare() {
+  wget "https://github.com/sass/libsass/archive/${pkg_version}.tar.gz" -O - | tar -xz
+}
+
+do_build() {
+  export SASS_LIBSASS_PATH="${HAB_CACHE_SRC_PATH}/${pkg_name}-${pkg_version}/libsass-${pkg_version}"
+  make
+}
+
+do_install() {
+  mv ./bin "${pkg_prefix}/"
```

Comment:
Prefer using [`install`(1)](https://linux.die.net/man/1/install) over mv or cp, as you can ensure it has the correct mode and attributes.